.circleci/config.yml

```diff
@@ -239,6 +232,9 @@ jobs:
             tar cjf pyodide-build-${CIRCLE_TAG}.tar.bz2  pyodide/
```

Comment:
I think it's better to leave "./" in the github release. People who download the release presumably want to serve their own version, and I think they are better off seeing a 404 than unexpectedly hitting the cdn.